src/sys/node/bundles/sys-util.js

```diff
@@ -1,4 +1,4 @@
-exports.turbocolor = require('turbocolor');
+exports.tc = require('turbocolor');
```

Comment:
Personally, i am fine with the PR, but this line.

In order to reduce the number of "dependencies", we bundle devDependencies in sysUtils (for faster installation), I think it makes sense to keep the full name of the package here.

Then, i am fine to use the `tc` alias.

```
const tc = this.tc = sysUtil.turbocolor;
```

Reply:
@manucorporat Sure. I changed this line and also the NodeLogger constructor to grab the library from `sysUtil.turbocolor`. Did I miss anything?